package.json

```diff
@@ -174,6 +174,7 @@
 		"test:php": "node ./tools/local-env/scripts/docker.js run -T php composer update -W && node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit",
 		"test:e2e": "node ./tests/e2e/run-tests.js",
 		"test:visual": "node ./tests/visual-regression/run-tests.js",
-		"wp-packages-update": "wp-scripts packages-update"
+		"wp-packages-refresh-deps": "node ./refresh-packages.js",
```

Comment:
What would be the use case to run this script in isolation? After every change to npm packages, you always need to run the build to ensure that all files get correctly updated in the `src/wp-includes` folder. There was a discussion to  remove most of those files coming from Gutenberg from the version control, but it hasn't happened yet.

Reply:
That path is no longer there, that's just a dev artifact. I removed it now.